.circleci/config.yml

```diff
@@ -71,7 +71,10 @@ jobs:
           key: *cache_key
       - run: bazel run @yarn//:yarn
       # Don't occupy the bazel server, as this test wants to run Bazel itself
```

Comment:
Add a TODO here to replace this shenanigans with a protractor rule when we have one.

Do you think there's enough speed advantage to run the -dev and -prod variants in parallel by making two jobs? probably not...